enum.rb

```diff
@@ -0,0 +1,53 @@
+#:nodoc:
+module Enumerable
+  def my_each
+    length.times { |index| yield(self[index]) }
+    self
+  end
+
+  def my_each_with_index
+    length.times { |index| yield(self[index], index) }
+    self
+  end
+
+  def my_select
+    array = []
+    my_each { |index| array << index if yield(index) }
+    array
+  end
+
+  def my_all?
+    new_array = true
+    my_each { |index| break new_array = false unless yield(index) }
+    new_array
+  end
+
+  def my_any?
```

Comment:
###  Should have working `my_any?`  identical and returns the same thing as ruby's [`any?`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-any-3F). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_any.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
``` p %w[ant bear cat].my_any?(/d/) ``` should return same results as ``` p %w[ant bear cat].any?(/d/) ```
``` p [nil, true, 99].my_any?(Integer)``` should return same results as ```p [nil, true, 99].any?(Integer)```
``` p [nil, true, 99].my_any? ``` shoud return same results as ```p [nil, true, 99].any?(Integer)```
``` p [12,3,34].my_any?(Integer)``` should return same results as ```p [nil, true, 99].any?(Integer)```